Command/IndexRebuildCommand.php

```diff
@@ -52,7 +60,33 @@ public function execute(InputInterface $input, OutputInterface $output)
                 ->setSQLLogger(null);
```

Comment:
@danrot still not sure that we should be doing this here -- it must be possible to disable DBAL logging when `%kernel.debug%` is false.